Article/Article.js

```diff
@@ -3,18 +3,20 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => {
+      this.expandArticle();
+    });
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.domElement.classList.toggle('article-open');
```

Comment:
you can change the expand button to collapse or close, after you open the article.  That would be good whenever you expand the article.  so you could do a this.expandButton.textContent = 'collapse'; inside of your expandArticle() method.  whenever an article is expanded.